### DISCUSSION TITLE
Bootstrap tauri code within rust app

### DISCUSSION THREAD

**Discussion by ArthurFleischman** [tauri-apps/tauri#7421] [Q&A]
Hi guys, I'm trying to create an monorepo app where every component of it runs asynchronously using Tokio, the Tauri+yew app is just one of those modules, how can i start the desktop/frontend part while maintaining the tauri app in the same runtime as my projetc? (aka, how can I start a tauri app within rust code)

**Comment by FabianLars** [comment]
Well, as long as you don't need any of the bundler features you really only need to care about the `tauri::Builder` stuff you see in the template projects. You can virtually execute that anywhere at any point in time as long as it's on the main thread (can be any thread if you don't need macos support)

No idea about the frontend though. And also not really for tauri's bundler/cli but i think i don't understand the idea here enough for that 🤔
